stylesheets/style.css

```diff
@@ -12,3 +12,15 @@ footer {
   min-width: 680px;
   margin: 0 auto;
 }
+
+.tutorial-filelist__list {
+}
+
+.hidden {
+	display: none;
+	visibility: hidden;
```

Comment:
But I do want to hide it from screen readers if the JS is available because the zip links will work and are accessible, right?